### ISSUE TITLE
Include titles

### ISSUE THREAD

**Issue by reagle** [zoni/obsidian-export#155]
Thank you for this; it's very fast! It'd be great to have an option for the resulting markdown pages to have titles, be it the first H1 `#` heading or in a yaml preface.

**Comment by langfingaz** [comment]
Do I understand it correctly that you would like to have a CLI option, that inserts the filename (without the .md suffix) as `# FILENAME` on top of the generated markdown output? (Or alternatively as key-value pair in the YAML preface?)

**Comment by reagle** [comment]
Yes, YAML title if present, otherwise file name without extension.
